README.md

```diff
@@ -174,14 +176,16 @@ for Intel IPU ES2K.
    # git checkout ipdk_v23.07
    ```
 
-3. Build the P4 Kubernetes binaries:
-  ```bash
-  # make build
-  ```
-  Then build the Kubernetes container images:
-  ```bash
-  # make docker-build
-  ```
+3. Build the P4 Kubernetes binaries. Note that, by default, Makefile is
+   configured to build for ES2K target. To build for P4-DPDK target, use
+   "tagname=dpdk" argument to following make targets.
+   ```bash
+   # make build
+   ```
+   Then build the Kubernetes container images:
```

Comment:
It has to be inline with previous comment for make. So can you please add tagname=dpdk for the docker image as well.